cloudslang-entities/src/test/java/io/cloudslang/lang/entities/DeserializeTest.java

```diff
@@ -1,33 +1,31 @@
 package io.cloudslang.lang.entities;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
-import com.google.gson.Gson;
 import io.cloudslang.lang.entities.bindings.Argument;
-import io.cloudslang.lang.entities.bindings.Result;
-import org.junit.Test;
 import io.cloudslang.lang.entities.bindings.Input;
 import io.cloudslang.lang.entities.bindings.Output;
+import io.cloudslang.lang.entities.bindings.Result;
+import org.junit.Before;
+import org.junit.Test;
 
 import java.io.IOException;
 
-import static org.junit.Assert.*;
+import static org.junit.Assert.assertEquals;
 
 public class DeserializeTest {
 
-    private ObjectMapper mapper = new ObjectMapper();
-    Gson gson = new Gson();
+    private ObjectMapper mapper;
 
+    @Before
```

Comment:
Do we really need it to be set up before each test case (Before annotation) or only once before running all the tests in the class (BeforeClass)?